module-generation/scrape.py

```diff
@@ -0,0 +1,67 @@
+import os
+import json
+import requests
+from github import Github
+
+indexDir = "./scrape-data/"
+os.mkdir(indexDir)
+
+accessToken = os.environ.get('GIT_TOKEN')
+g = Github(accessToken)
+
+user = g.get_user('Terasology')
+repo = user.get_repo("Index")
+allModules = repo.get_contents("/distros/omega/gradle.properties")
+getModules = allModules.decoded_content.decode()
+for line in getModules.split():
+    if  line.startswith('extraModules='):
+        filterData=line[13:]
+        modules=filterData.split(",")
+
+# Collect modules 
+for module in range(len(modules)):
+    if(module<len(modules)):
```

Comment:
what exactly do you need this condition for?

Reply:
oh, I forgot to remove that, I don't have an access token so I test it for 2 to 3 modules so that  I do not exceed the limit of api calls,Removed